packages/node_modules/@ciscospark/widget-recents/src/container.js

```diff
@@ -638,14 +676,24 @@ export class RecentsWidget extends Component {
       });
 
       let filteredSpacesListWithActivityText;
+      // Filter space list based on type
+      const groupBy = spaceTypeFilter === SPACE_TYPE_GROUP || spaceTypeFilter === SPACE_TYPE_ONE_ON_ONE;
+      if (groupBy) {
+        filteredSpacesListWithActivityText = this.constructor.filterSpace({
```

Comment:
FYI, to call the static function, use `RecentsWidget.filterSpace` instead.

Reply:
I've moved all filter-related functions into selector